src/utilities/__tests__/findBreakingChanges-test.js

```diff
@@ -392,6 +392,54 @@ describe('findBreakingChanges', () => {
     ]);
   });
 
```

Comment:
This test asserts were not using instance equality to validate that an argument has changed in a breaking way (this test will fail if so)